docs/actions/http.md

```diff
@@ -1,58 +1,25 @@
-# HTTP
+# GET
 
-## Global Notes
+=== "Inputs"
 
-### Security
-  All the HTTP tasks uses the same security logic.
-  It is defined via the **target** input properties :
+    | Required | Name      | Type                 |  Default   |
+    |:--------:|:----------|:---------------------|:----------:|
+    |    *     | `target`  | String               |            |
+    |    *     | `uri`     | String               |            |
+    |          | `headers` | Map<String, String\> |            |
+    |          | `timeout` | String               | "2000 ms"  |
```

Comment:
It could be nice to have all value possible to timeout. May be add a common section and link timeout to it.